lira/lira_utils.py

```diff
@@ -19,6 +21,26 @@
 LIRA_SERVER_HEADER = {'Server': 'Lira Service', 'Content-type': 'application/json'}
 
 
+def _is_authenticated_pubsub(request):
+    """ Check if the message is sent by Google:
+    https://google-auth.readthedocs.io/en/latest/reference/google.oauth2.id_token.html """
+    try:
+        # Get the Cloud Pub/Sub-generated JWT in the "Authorization" header.
+        bearer_token = request.headers.get('Authorization')
+        token = bearer_token.split(' ')[1]
+
+        # Verify and decode the JWT. `verify_oauth2_token` verifies
+        # the JWT signature, the `aud` claim, and the `exp` claim.
+        claim = id_token.verify_oauth2_token(token, google_transport_requests.Request())
+        # Must also verify the `iss` claim.
+        if claim['iss'] not in ['accounts.google.com', 'https://accounts.google.com']:
+            logger.error('Wrong issuer.')
+            return False
+        return True
+    except Exception as e:
+        logger.error('Invalid token: {}\n'.format(e))
```

Comment:
nit:
```suggestion
        logger.error(f'Invalid token: {e}\n')
```
for consistency